holmes/core/llm.py

```diff
@@ -217,6 +220,10 @@ def completion(
         if self.args.get("thinking", None):
             litellm.modify_params = True
 
+        if REASONING_EFFORT:
```

Comment:
Consider add comment that when upgrading litellm we can remove that.

Reply:
thanks, it automerged after your approval so I'll fix separately